qui/devices/device_widget.py

```diff
@@ -90,32 +99,38 @@ def __init__(self, app_name, qapp, dispatcher):
         self.initialize_dev_data()
 
         for devclass in DEV_TYPES:
-            self.dispatcher.add_handler('device-attach:' + devclass,
-                                        self.device_attached)
-            self.dispatcher.add_handler('device-detach:' + devclass,
-                                        self.device_detached)
-            self.dispatcher.add_handler('device-list-change:' + devclass,
-                                        self.device_list_update)
-
-        self.dispatcher.add_handler('domain-shutdown',
-                                    self.vm_shutdown)
-        self.dispatcher.add_handler('domain-start-failed',
-                                    self.vm_shutdown)
-        self.dispatcher.add_handler('domain-start', self.vm_start)
-
-        self.dispatcher.add_handler('property-set:template_for_dispvms',
-                                    self.vm_dispvm_template_change)
-
-        self.dispatcher.add_handler('property-reset:template_for_dispvms',
-                                    self.vm_dispvm_template_change)
-        self.dispatcher.add_handler('property-del:template_for_dispvms',
-                                    self.vm_dispvm_template_change)
+            self.dispatcher.add_handler(
+                "device-attach:" + devclass, self.device_attached
+            )
+            self.dispatcher.add_handler(
+                "device-detach:" + devclass, self.device_detached
+            )
+            self.dispatcher.add_handler(
+                "device-list-change:" + devclass, self.device_list_update
+            )
+
+        self.dispatcher.add_handler("domain-shutdown", self.vm_shutdown)
+        self.dispatcher.add_handler("domain-start-failed", self.vm_shutdown)
+        self.dispatcher.add_handler("domain-start", self.vm_start)
+
+        self.dispatcher.add_handler(
+            "property-set:template_for_dispvms", self.vm_dispvm_template_change
+        )
+
+        self.dispatcher.add_handler(
+            "property-reset:template_for_dispvms",
+            self.vm_dispvm_template_change,
+        )
+        self.dispatcher.add_handler(
+            "property-del:template_for_dispvms", self.vm_dispvm_template_change
+        )
 
         self.widget_icon = Gtk.StatusIcon()
-        self.widget_icon.set_from_icon_name('qubes-devices')
-        self.widget_icon.connect('button-press-event', self.show_menu)
+        self.widget_icon.set_from_icon_name("qubes-devices")
```

Comment:
While you are at it, it might be a good opportunity to take a loot at issue [#9208](https://github.com/QubesOS/qubes-issues/issues/9208) as well. Workaround should be as easy and saving the `qubes-devices` svg file as a 256x256 svg. Or maybe providing the icon as PNG files of different sizes as well. I do not have i3wm installed and could not verify the outcome myself.

Reply:
I cannot reproduce this issue because I don't really see what's wrong. For me, the domain widget tray icon is exactly the same than other icons even if I increase the bar size.